### PR TITLE
Creates a webapp for creating config files

### PR DESCRIPTION
**Pull request by utkbansal**
* Creates a basic form to create the Supernova config

### CODE REVIEW

**Comment by utkbansal** [comment]
Why does it say that it could not get flask-wtf? It works fine on my  system.

**Comment by wkerzendorf** [comment]
it should work with anaconda, that is what is currently used for travis. See what anacondas package for flask is.

**Comment by utkbansal** [comment]
it is 'flask' as mentioned here - http://docs.continuum.io/anaconda/pkg-docs.html

**Comment by wkerzendorf** [comment]
then you need to change the pip-requirements file (better yet rename it to `conda_requirements.txt` and change the `.travis.yml` file

File: conda-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This file may be used to create an environment using:
```
Comment (wkerzendorf) [inline conda-requirements.txt]: just use the old `pip-requirements` file for this and put the things that don't exist in anaconda into the `pip-requirements` file

Comment (wkerzendorf) [inline conda-requirements.txt]: i  see you changed this. one last thing get rid of any py27_0

**Pull request closed without merging**